shared/chat/conversation/messages/wrapper/index.tsx

```diff
@@ -250,7 +250,10 @@ class _WrapperMessage extends React.Component<Props & Kb.OverlayParentProps, Sta
   _isFailed = () =>
     !!this.props.failureDescription && (
       <Kb.Text key="isFailed" type="BodySmall">
-        <Kb.Text type="BodySmall" style={styles.fail}>
+        <Kb.Text type="BodySmall" style={this._isExploding() ? styles.failExploding : styles.fail}>
+          {this._isExploding() && (
+            <Kb.Icon fontSize={16} boxStyle={styles.failExplodingIcon} type="iconfont-block" />
+          )}{' '}
```

Comment:
Does this insert a space when `this._isExploding() === false`?

Reply:
used some padding instead!